Project.toml

```
@@ -4,6 +4,7 @@ authors = ["Aaron Peikert", "Maximilian S. Ernst", "Clifford Bode", "Nicklas Haf
 version = "0.1.0"
 
 [deps]
+AbstractTrees = "1520ce14-60c1-5f80-bbc7-55ef81b5835c"
```

```suggestion
AbstractTrees = "1520ce14-60c1-5f80-bbc7-55ef81b5835c"
```

This should not be on the toplevel, but under docs/Project.toml right?